multichain/openrpc.yaml

```diff
@@ -62,6 +62,80 @@ methods:
           value: true
     errors:
       - $ref: "#/components/errors/UnknownError"
+  - name: wallet_sessionChanged
+    paramStructure: by-name
+    description: >-
+      This notification is published by the wallet to notify the callers of updates to a shared session's authorization scopes.
+      The event payload contains the new `sessionScopes`. If a connection between the wallet and the caller is severed
+      and the possibility of missed events arises, the caller should immediately call `wallet_getSession` to retrieve
```

Comment:
I think we can remove the `If a connection between the wallet and the caller is severed...` onward since it doesn't apply to our wallet?